vendor/deps/cli-kit/lib/cli/kit/resolver.rb

```diff
@@ -29,6 +29,17 @@ def command_not_found(name)
           $stderr.puts(CLI::UI.fmt("{{command:#{@tool_name} #{name}}} was not found"))
         end
 
+        if ShopifyCli::Project.has_current?
+          $stderr.puts(CLI::UI.fmt(<<~INFO
```

Comment:
We probably want to add these messages to `messages.rb` if we can.

Reply:
I don't believe this is possible, because we may not have `@cxt`.

Reply:
You can always create a new context object, getting messages doesn't rely on any settings inside context (which may mean we should move it out of context at some point!).